system/Test/CIDatabaseTestCase.php

```diff
@@ -135,6 +142,8 @@ public function __construct()
 	 */
 	public function setUp()
 	{
+		$this->loadDependencies();
```

Comment:
Can you describe the use-case for having it run prior to every test instead of just loading once per test file? At first, I thought that it would allow specifying different seeds, etc before a test but that doesn't work since it's in the setup phase which happens prior.